docs/manual/getting_started/superdesk-superdesk-publisher-setup.rst

```diff
@@ -1,403 +1,113 @@
-    && sudo -u www-data SYMFONY_ENV=prod composer install --no-dev --optimize-autoloader
-
-All the source code dependencies will start to install.
-Once it is done, you will be asked to fill the ``parameters.yml`` file which needs to be completed before proceeding.
-
-If you don’t know what to set, just simply use default values by hitting “enter”
-and replace the content of ``/var/www/publisher/app/config/parameters.yml`` file with:
-
-.. code-block:: yaml
-
-    # This file is auto-generated during the composer install
-    parameters:
-        env(DATABASE_HOST): 127.0.0.1
-        env(DATABASE_PORT): null
-        env(DATABASE_NAME): publisher
-        env(DATABASE_USER): postgres
-        env(DATABASE_PASSWORD): postgres
-        env(DATABASE_SERVER_VERSION): 9
-        mailer_transport: smtp
-        mailer_host: 127.0.0.1
-        mailer_user: null
-        mailer_password: null
-        env(SYMFONY_SECRET): SuperSecretTokenPleaseChangeIt
-        swp_updater.version.class: SWP\Bundle\CoreBundle\Version\Version
-        env(SWP_DOMAIN): 192.168.0.102 # server domain/IP where Superdesk Publisher is installed
-        cache_servers:
-            - 192.168.0.102 # server domain/IP where Superdesk Publisher is installed
-        doctrine_cache_driver: array
-        sentry.dsn: false
-        session_memcached_host: localhost
-        session_memcached_port: 11211
-        session_memcached_prefix: sess
-        session_memcached_expire: 3600
-        allow_origin_cors: '*'
-        superdesk_servers:
-            - 192.168.0.101 # server domain/IP where Superdesk is installed
-        env(ELASTICA_HOST): localhost
-        env(ELASTICA_PORT): 9200
-        env(RABBIT_MQ_HOST): 127.0.0.1
-        env(RABBIT_MQ_PORT): 5672
-        env(RABBIT_MQ_USER): guest
-        env(RABBIT_MQ_PASSWORD): guest
-
-And set proper permissions for ``cache`` and ``logs`` directories, run:
-
-.. code-block:: bash
-
-    sudo setfacl -dR -m u:"$HTTPDUSER":rwX -m u:$(whoami):rwX app/cache app/logs
-    && sudo setfacl -R -m u:"$HTTPDUSER":rwX -m u:$(whoami):rwX app/cache app/logs
-
-3. Check Requirements
----------------------
-
-Check if your server meets the requirements by running:
-
-.. code-block:: bash
-
-    php app/check.php
-
-If everything is in order, you should see this message: ``Your system is ready to run Symfony projects on your screen.``
+Superdesk Publisher Component is a JavaScript component that is a separate dependency
+and can be included in Superdesk to manage your Superdesk Publisher application.
 
-4. Create the Database and Update the Schema
---------------------------------------------
+To add this component as a dependency edit ``/opt/superdesk/client/package.json`` and include the 
+Superdesk Publisher repository:
 
-Inside ``/var/www/publisher`` directory, run the command to create the database:
+.. code-block:: js
 
-.. code-block:: bash
+	"dependencies": {
+		....,
+		"superdesk-publisher": "superdesk/superdesk-publisher#master"
```

Comment:
how about changing it to `superdesk/superdesk-publisher#2.0` ? Master likes to be unstable ;)